app/script/view_model/content/InputBarViewModel.js

```diff
@@ -138,6 +138,18 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
         .join('');
     });
 
+    this.richTextInput.subscribe(() => {
+      const textarea = document.querySelector('#conversation-input-bar-text');
```

Comment:
We have the ID `#conversation-input-bar-text` now in three different places. Let's do a constant for it, so we can update it in a single place.